src/pane-container.coffee

```diff
@@ -15,6 +15,8 @@ class PaneContainer extends Model
   @version: 1
 
   root: null
+  stoppedChangingActivePaneItemDelay: 100
```

Comment:
100ms is arbitrary. `TextEditor::didStopChanging` uses 300ms, which seemed too long here but for no measured reason. Does 100ms seem too short or too long?